features/jupyter.feature

```diff
@@ -10,10 +10,12 @@ Feature: Jupyter targets in new project
 
   Scenario: Execute jupyter notebook target
     When I execute the kedro jupyter command "notebook --no-browser"
+    Then I wait for the jupyter webserver to run for up to "360" seconds
```

Comment:
How did you decide on 360 seconds? Does it generally take that long for the jupyter webserver to startup? 😮

Reply:
It's arbitrary my reasoning was just to give it 5 minutes in case CI performs slow one day, in actuality, it runs the server pretty quickly, within 30 seconds.

Reply:
My suggestion would be to lower the timeout in that case. If it usually runs within 30 secs, 2 mins timeout is already quite generous. We can always up it again in case it does take longer on CI generally.